### DISCUSSION TITLE
Converting a Vue 2 App to Alpine.js

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#18] [Comments]
# Converting a Vue 2 App to Alpine.js

A look at a conversion from Vue 2 to Alpine.js

https://www.raymondcamden.com/2024/03/04/converting-a-vue-2-app-to-alpinejs

**Comment by robertz** [comment]
I almost forgot how addicting IdleFleet was, I have been playing the heck out of it the last few days. I appreciate you outlining the differences between the Vue and Alpine versions of the app. I have been wanting to pick up Alpine and this helped me get up to speed.
